docs/source/model_design/risk_factors.rst

```diff
@@ -195,7 +195,118 @@ relationships directionally between a cause/exposure and effect/outcome.
   Discuss criteria (sim science and GBD) for modeling a risk outcome pair
 
 Confounding
-""""""""""""
+"""""""""""
+**What is confounding?**
```

Comment:
It looks like your confounding documentation accidentally made it into this PR! You might want figure out how to take it out so that it doesn't cause any conflicts with your confounding branch/PR.

Reply:
grrr...how do I get rid of it?

Reply:
You need to check out the master branch, do git pull to make sure it's updated, and branch from master instead - you probably accidentally branched from your confounding branch (or just continued editing on that branch).

Reply:
ah that must be it! thanks :) noted for next time.